### PR TITLE
Feat add translation latin

### PR DESCRIPTION
**Pull request by fernandoamz**
## What does this PR do?
Add translation and files to load Spanish Lat.

## Test Plan
Someone who can run the project should make a checkout to review if the files and translations for latin are running correctly.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

This PR add the feature from this issue.
https://github.com/appwrite/appwrite/issues/1746

### CODE REVIEW

**Comment by fernandoamz** [comment]
I fixed the last comment @christyjacob4, thanks for your feedback

File: app/config/locale/codes.php [hunk 1]
```diff
@@ -37,9 +37,15 @@
     'kn', // Kannada
     'km', // Khmer
     'ko', // Korean
+<<<<<<< HEAD
```
Comment (christyjacob4) [inline app/config/locale/codes.php:40]: You have committed the code while resolving a merge conflict. Please fix this

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

**Comment by fernandoamz** [comment]
Hi @christyjacob4 , I fixed the requested changes

File: app/config/locale/codes.php [hunk 2]
```diff
@@ -37,8 +37,10 @@
     'kn', // Kannada
     'km', // Khmer
     'ko', // Korean
+    'la', // Latin
     'lb', // Luxembourgish
     'lt', // Lithuanian
+    'la', // Latin
```
Comment (christyjacob4) [inline app/config/locale/codes.php:43]:
```suggestion
```
We still have 2 occurrences for the Latin language. Can you remove this one. ?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please address one last comment. Rest looks good 👍

**Comment by fernandoamz** [comment]
Thanks so much @christyjacob4, I fixed the last comment.

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+is%3Aissue+label%3Ahacktoberfest+created%3A%3E2021-01-01+is%3Aopen&type=issues) and help make Appwrite better :)

**Pull request merged**